flight/Modules/Stabilization/inc/virtualflybar.h

```diff
@@ -43,4 +43,4 @@ int stabilization_virtual_flybar_pirocomp(float z_gyro, float dT);
 /**
  * @}
  * @}
- */
+ */
```

Comment:
5.1.1.2 of ISO C 1999, see here for other reasons (like breaking various unix utils): http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline